src/data/federal/2017/SNAPData.js

```diff
@@ -1,10 +1,24 @@
-/*
- * RESOURCES
- * Derived from https://www.masslegalservices.org/content/online-snap-calculator
- * http://www.mass.gov/eohhs/consumer/basic-needs/financial/program-eligibility-charts-and-tables.html
- * http://www.mass.gov/eohhs/gov/departments/dta/program-eligibility-charts-and-tables.html
+/** Data tables for MA SNAP calculations.
+ *
+ * RESOURCES:
+ * Derived from {@link https://www.masslegalservices.org/content/online-snap-calculator}
+ * See more resources in the actual code since some
+ *      can't be referenced in these comments.
+ *
+ * @namespace
  */
 const SNAPData = {
+  // Code-style note: Properties have to use quotes
+  /* MORE RESOURCES
+   * (Since pdfs and htmls can't seem to be referenced safely in jsdoc comments)

```

Comment:
I think you can put them in the docblock as long as it's before the `@namespace` tag

Reply:
See below.